icevision/models/torchvision_models/faster_rcnn/prediction.py

```diff
@@ -70,3 +78,12 @@ def convert_raw_prediction(raw_pred: dict, detection_threshold: float):
         "bboxes": bboxes,
         "above_threshold": above_threshold,
     }
+
+
+def get_preds(model: nn.Module, dataset: Dataset, batch_size: int = 16) -> List[dict]:
```

Comment:
I get your point.
I added it just to avoid writing separately the 2 lines which the function contains.
We can keep it and make it private or I can just simply type 
```
dl = infer_dl(dataset, batch_size=batch_size)
samples, preds = predict_dl(model=model, infer_dl=dl)
```
Which one do you prefer?

Reply:
Is this function only called in one place? If so I think it's better to just have the two lines

Reply:
Ok will do that then. Yes, it is called in just one place.

Reply:
Done.